hcfb-be/backend-server/app/controller/distribution.controller.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../config/db.config.js');
+const Distribution = db.distributions;
+
+exports.create = (req, res) => {
+    //Save to postgre database
```

Comment:
```suggestion
    //Save to postgresql database
```

Comment spelling 👨‍🎨